assets/src/components/conversations/ConversationHeader.tsx

```diff
@@ -112,23 +116,40 @@ const ConversationHeader = ({
             )}
           </Box>
 
-          <Box mx={1}>
-            {status === 'closed' ? (
-              <Tooltip title="Reopen conversation" placement="bottomRight">
-                <Button
-                  icon={<UploadOutlined />}
-                  onClick={() => onReopenConversation(conversationId)}
-                />
-              </Tooltip>
-            ) : (
+          {status === 'closed' ? (
+            <Fragment>
+              <Box mx={1}>
+                <Tooltip title="Reopen conversation" placement="bottomRight">
+                  <Button
+                    icon={<UploadOutlined />}
+                    onClick={() => onReopenConversation(conversationId)}
+                  />
+                </Tooltip>
+              </Box>
+              <Box mx={1}>
+                <Popconfirm
+                  title="Are you sure"
```

Comment:
can we change this to "Are you sure you want to delete this conversation?"

just want to make it as clear as possible what's going on :)